docs/services/invocation.md

```diff
@@ -141,38 +141,60 @@ The Invocation identifier is opaque and its current format should not be relied
 
 ## Cancel an invocation
 
-:::caution
+If an invocation takes too long to complete and, therefore, is no longer of interest, you can cancel it.
+Canceling an invocation allows it to free any resources it is holding and roll back any changes it has made so far.
+In order to roll back changes correctly, the service handlers need to contain the necessary compensation logic.
+If the required compensation logic is implemented, then the service state stays consistent even in the presence of cancellations.
```

Comment:
Do the docs explain somewhere how to implement compensation logic? Might be good to add otherwise?